app/modules/integrations/integrations_router.py

```
@@ -1226,6 +1476,263 @@ async def jira_webhook(
         )
 
 
+def verify_jira_webhook_jwt(
```

Ruff (0.14.3)

Redefinition of unused `verify_jira_webhook_jwt` from line 1222

(F811)